src/styles/api-workflows.css

```
@@ -0,0 +1,744 @@
+    z-index: 1000;
+    justify-content: center;
+    align-items: center;
+    padding: 1.25rem;
+}
+
+.prompt-modal.show {
+    display: flex;
+    animation: pfadeIn 0.3s ease;
+}
+
+@keyframes pfadeIn {
```

Stylelint (17.7.0)

[error] Expected keyframe name "pfadeIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)